lib/note-editor.jsx

```diff
@@ -76,19 +71,19 @@ export const NoteEditor = React.createClass( {
 
 	onCancelRevision: function() {
 		// clear out the revision
-		this.setState( { revision: null } );
+		this.props.onRevisionChange( { revision: null } );
 		this.setIsViewingRevisions( false );
 	},
 
-	setIsViewingRevisions: function( isViewing ) {
-		this.setState( { isViewingRevisions: isViewing } );
+	setIsViewingRevisions: function( isViewingRevisions ) {
+		this.props.onRevisionChange( { isViewingRevisions } );
 	},
```

Comment:
Definitely trimmable.
But about the one argument: that won't be an issue anymore once I separate the various functions included in `onRevisionChange`.